apps/api/src/common/hotcache.ts

```diff
@@ -0,0 +1,22 @@
+class CacheObject {
+    constructor(public value: any, public expiration: number) {}
+    isExpired() {
+        return this.expiration < Date.now()
+    }
+}
+
+export class HotCache {
+    private cache: { [key: string]: CacheObject } = {}
```

Comment:
Since we are going to deal with many insertions/deletions it is better to use [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), instead of regular objects

Reply:
Actually we currently have only 3 records, so no more than 3 insertions are expected.
The value will be updated every 30 seconds, but no deleting is planned.

Reply:
Just read that it can be a security issue to use an object here. 
Map is safer for keys that can come from user input.
Changed it.